examples/profiling/README.md

```diff
@@ -1,59 +1,91 @@
 # Profiling guest code
 
-In this example we demonstrate how to profile different implementations of the Fibonacci sequence calculation on the RISC0 ZKVM.
+In this example we demonstrate how to profile on the RISC Zero zkVM guest programs, using the Fibonacci sequence calculation as an example.
 
-## Overview
+<!-- NOTE: This content matches the docs under website/api/zkvm/developer-guide/profiling.md -->
+## Background
 
-There are three different Fibonacci sequence calculation methods provided:
+Profiling tools, like [pprof] and [perf], allow collecting performance information over the entire execution of your program, and help create visualizations for the performance of your program.
+RISC Zero has experimental support for generating pprof files for cycle counts.
 
-1. `fibonacci_1`: A basic iterative method.
-2. `fibonacci_2`: An optimized iterative method that attempts to batch computation.
-3. `fibonacci_3`: A matrix exponentiation approach, which is a fast method to compute Fibonacci numbers.
-
-The host code sets up a profiling environment and then runs the RISC0 guest code that computes Fibonacci numbers using the three methods. After execution, the profiling data is written out for further analysis.
+[Sampling CPU profilers], as implemented by pprof and perf, provide a view of where your program is spending its time.
+It does so by recording the current call stack at a sampling interval.
+RISC Zero provides a "sampling" [^1] CPU profiler for guest execution.
 
-The guest code reads the number of iterations from the host, computes the Fibonacci number, and finally commits the answer back to the host.
+[pprof]: https://github.com/google/pprof
+[perf]: https://perf.wiki.kernel.org/index.php/Main_Page
+[Sampling CPU profilers]: https://nikhilism.com/post/2018/sampling-profiler-internals-introduction/
 
 ## Usage
 
 ### Step 1: Prerequisites
+
 First, [install Rust] if you don't already have it.
+Additionally, you will need to [install Go], which bundles with it the [pprof] tool.
 
 Next, install the `cargo-risczero` tool and install the toolchain with:
+
 ```bash
 cargo install cargo-binstall
 cargo binstall cargo-risczero
 cargo risczero install
 ```
 
-Finally, install the [pprof] tool with
-```bash
-go install github.com/google/pprof@latest
-```
-
 ### Step 2: Running
-Then, run the example with:
+
+Run the Fibonacci profiling example with:
+
 ```bash
-RISC0_PPROF_OUT=./profile.pb cargo run --release
+RISC0_PPROF_OUT=./profile.pprof cargo run
```

Comment:
Is there a reason to go with the extension .pprof? I originally used .pb as it is the same extension (aside from being .gz) used by pprof (https://github.com/google/pprof#generate-a-graph-in-an-svg-file-and-open-it-with-a-web-browser)

Reply:
I just thought it sounded good. I'll go with `.pb` instead 👍 
9e8f10032be5b38e6dde7d2cf8c53fed4b8b52a9